crowdin.yaml

```diff
@@ -0,0 +1,7 @@
+files:
+  -
+    source: '/resources/lang/en/*.php'
```

Comment:
This will allow us to push changes to the English translation and have them synced. Nice.